application-admintools-api/src/main/java/com/xwiki/admintools/ServerIdentifier.java

```diff
@@ -65,4 +67,11 @@ public interface ServerIdentifier
      * Update the possible paths to the configuration files.
      */
     void updatePossiblePaths();
+
+    /**
+     * Access the Active Installs 2 Ping to retrieve server name and version.
```

Comment:
You don't need to specify the Active Installs 2 Ping in here. Maybe add a comment inside the method
Also, you might need other data besides name and version in the future, so you can say `metadata` to cover that